pkg/applicationserver/io/packages/loradms/v1/api/objects/objects.go

```diff
@@ -79,14 +81,29 @@ func (d DeviceUplinkResponses) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// PositionSolution is the solution of a position calculation.
+type PositionSolution struct {
+	ECEF           []float64 `json:"ecef"`
+	LLH            []float64 `json:"llh"`
+	CaptureTimeGPS float64   `json:"capture_time_gps"`
+	GDOP           float64   `json:"gdop"`
+	Accuracy       float64   `json:"accuracy"`
+	Timestamp      float64   `json:"timestamp"`
+}
+
 // UplinkResponse contains the state changes and completed items due to an uplink message.
 type UplinkResponse struct {
-	File              *File            `json:"file"`
-	StreamRecords     []Stream         `json:"stream_records"`
-	FullfiledRequests []PendingRequest `json:"fulfilled_requests"`
-	Downlink          *LoRaDnlink      `json:"dnlink"`
-	InfoFields        InfoFields       `json:"info_fields"`
-	LogMessages       []LogMessage     `json:"log_messages"`
+	File                  *File             `json:"file"`
+	StreamRecords         []Stream          `json:"stream_records"`
+	PositionSolution      *PositionSolution `json:"position_solution"`
+	PendingRequests       *PendingRequests  `json:"pending_requests"`
+	FullfiledRequests     []Request         `json:"fulfilled_requests"`
+	Downlink              *LoRaDnlink       `json:"dnlink"`
+	InfoFields            InfoFields        `json:"info_fields"`
+	LogMessages           []LogMessage      `json:"log_messages"`
+	UploadedFiles         []File            `json:"uploaded_files"`
+	UploadedStreamRecords []Stream          `json:"uploaded_stream_records"`
+	LastUplink            *LoRaUplink       `json:"last_uplink"`
 }
```

Comment:
That is a `DEVICEINFO` structure, not an `UPLINK_RESPONSE`  
https://www.loracloud.sm.tc/documentation/device_management?url=v1.html#uplinkresponse  
https://www.loracloud.sm.tc/documentation/device_management?url=v1.html#device-info